stacker_blueprints/rds/aurora/base.py

```diff
@@ -179,6 +184,18 @@ def create_security_group(self):
                 )
             )
             self.security_group = Ref(sg)
+
+            if variables["AllowedCIDRs"]:
```

Comment:
You have a good point, this isn't really that flexible. Would you find the TroposphereType to be a good middle ground, as you mentioned in the other issue? I find having seperate stacks for the SGs a bit cumbersome and usually avoid it.